scripts/measure-depth-at-tc.py

```
@@ -0,0 +1,273 @@
+    commands = "uci\n"
+    commands += "setoption name Threads value %d\n" % threads
+    commands += "setoption name Hash value 256\n"
+    commands += "isready\n"
+
+    for name, fen in POSITIONS:
+        commands += "position fen %s\n" % fen
+        commands += "go movetime %d\n" % movetime_ms
+
+    commands += "quit\n"
+
+    result = subprocess.run(
```

Ruff (0.15.5)

[error] `subprocess` call: check for execution of untrusted input

(S603)